README.md

```diff
@@ -17,7 +16,7 @@ Additional description about the project and its features.
 
 ## Live Demo
 
-[Live Demo Link](https://vikipretium.github.io/portfolio_project/)
+[Live Demo Link](https://vikipretium.github.io/)
 
```

Comment:
- [ ] In your README, kindly add instructions on how users can set up and run your app locally.

Reply:
fixed it !